default.py

```diff
@@ -146,9 +147,6 @@ def get_params():
 elif mode == 117:
     Radio.ListFavourites(logged_in)
 
-elif mode == 118:
```

Comment:
Do I understand this correctly: There is only "Listen Live" now, no more HQ streams? Or have they been integrated to "Listen Live"